server/middlewares/userValidation.js

```diff
@@ -123,23 +127,50 @@ class UserValidation {
    * @param {function} next - The Next Function
    */
   static async validateStatus(req, res, next) {
-    let error;
-    const validate = await Helpers.validate();
     const { email } = await req.params;
     const { status } = await req.body;
-    const arr = ['verified', 'unverified'];
-    const user = await Helpers.findByMail(email, users);
+    const arr = ['verified'];
+    const user = await db.query(getUserByEmail, [email]);
+
+    // check if the request body is correct
+    if (!arr.includes(status)) {
+      return res
+        .status(400)
+        .json({ status: 400, error: 'The status can either be verified or unverified' });
```

Comment:
this message is confusing `'The status can either be verified or unverified'`

Reply:
Its an error message to let the admin know that a user can only be verified or unverified, if he tries to pass another string to the request body.